container_pipeline/utils.py

```diff
@@ -33,27 +33,45 @@ def load_yaml(yaml_file):
     return data
 
 
-def run_cmd(cmd, check_call=True, no_shell=False):
+def run_cmd(cmd, check_call=True, no_shell=False, use_shell=False,
```

Comment:
why repeat the utilities code? check `container-pipeline-service/container_pipeline/lib/command.py` add necessary options in that utility function with proper defaults (to keep up with other code already using it)

Reply:
That is something we need to do, consolidate into one lib. We have various run_cmd and other such funcs scattered accross the board, which i too dont like. Maybe a seperate PR is needed.